docs/guide/breakpoint-preview.md

```diff
@@ -84,100 +84,95 @@ Each of the individual breakpoint properties in the `screen` object take a `labe
+  @media (600px <= width <= 1200px) { ... }
   ```
-  These operators are also fine in the context of a descriptor that accepts a range:
-  ```css
-    @media ( width >= 600px) { ... } /* Works fine */
 
 #### 3. **Behavioral Differences**
 Media queries operate on the **viewport size**, whereas container queries respond to the **size of a specific container**. This distinction can lead to subtle differences in layout and behavior:
+
 - **Viewport context**:
-  - Media queries are global; they consider the entire screen or browser window. If a layout condition depends on the viewport, such as full-page navigation menus, it might not behave as expected when converted to container queries.
+  - Media queries are global; they consider the entire screen or browser window. If a layout condition depends on the viewport, such as full-page navigation menus, it might not behave identically when converted to container queries.
+  
 - **Nested containers**:
   - Container queries work within the bounds of their parent container. If a nested container has a different size, the styles applied to its content may differ from expectations.
+  
 - **Resizing effects**:
-  - Breakpoint preview resizes containers but does not simulate the behavior of the entire viewport. This can impact layouts that rely on viewport-relative units like `vw` and `vh`.
-
-#### 3. **Specificity and Logical Operators**
-Complex media query conditions may not translate exactly when converted to container queries:
-- **Logical operators**:
-  - Combining multiple conditions with `and`, `or`, or `not` might result in unexpected behavior. For example:
-    ```css
-    @media (min-width: 600px) and (max-width: 800px) { ... }
-    ```
-    Container queries handle these operators differently and may not correctly apply styles in nested containers.
-- **Specificity conflicts** 
-  - Container queries may conflict with or override existing media queries or other styles due to CSS specificity rules. This is especially true when mixing global media queries and container-specific styles.
+  - Breakpoint preview resizes containers but does not simulate the entire viewport behavior. This can impact layouts that rely on viewport-relative units like `vw` and `vh`.
 
 #### 4. **Viewport-relative Units**
-- Units like `vw`, `vh`, `vmin`, and `vmax` are **not container-aware**. They continue to calculate values based on the viewport, not the container.
-  - Example of affected styles:
-    ```css
-    @media (min-width: 800px) {
-      .example {
-        width: 50vw; /* Still based on viewport width */
-      }
-    }
-    ```
+Units like `vw`, `vh`, `vmin`, and `vmax` are converted to their container-query equivalents (`cqw`, `cqh`, `cqmin`, `cqmax`), but they calculate values differently:
+
+```css
+/* Original CSS */
+.element {
+  width: 50vw; /* 50% of viewport width */
+}
+
+/* Converted to */
+.element {
+  width: 50cqw; /* 50% of container width, not viewport */
```

Comment:
Does this have any practical pitfalls? Reading about it, it seems like it should be fine?

Reply:
I updated this to better describe the pitfalls.